helpers/lib_tasks_docker_release.py

```diff
@@ -419,43 +624,17 @@ def docker_tag_push_multi_build_local_image_as_dev(  # type: ignore
     :param container_dir_name: directory where the Dockerfile is located
     """
     hlitauti.report_task(container_dir_name=container_dir_name)
-    #
-    hlitadoc.docker_login(ctx, target_registry)
-    #
-    dev_version = _get_dev_version(version, container_dir_name)
-    image_versioned_local = hlitadoc.get_image(
-        local_base_image, "local", dev_version
-    )
-    _LOG.info(
-        "Pushing the local dev image %s to the target_registry %s",
-        image_versioned_local,
+    _docker_tag_and_push_multi_arch_image(
+        ctx,
+        version,
+        local_base_image,
         target_registry,
+        container_dir_name,
+        stage="local",
```

Comment:
Need to change as in the last update.

Check everywhere.

Reply:
Although the `source_stage` parameter is positional, let's use the same variable name as in the function interface.